### ISSUE TITLE
Should we add the `enabled` option for add_index_field?

### ISSUE THREAD

**Issue by mejackreed** [projectblacklight/spotlight#1861]
For search and sort fields we have the ability to disable these for exhibits by default. Should/could we add the same for index/show fields?

https://github.com/projectblacklight/spotlight/pull/1851

**Comment by cbeer** [comment]
Yes, but it's a little trickier than just `enabled`. Facets and index/show fields are also view-type specific. Here's the equivalent code:

https://github.com/projectblacklight/spotlight/blob/master/app/models/spotlight/blacklight_configuration.rb#L315-L334
